TrkDiag/src/BkgDiag_module.cc

```diff
@@ -138,7 +140,9 @@ namespace mu2e
     _bkghToken{ consumes<BkgClusterHitCollection>(config().BkgClusterHitCollection() ) },
     _mcdigisToken{ consumes<StrawDigiMCCollection>(config().StrawDigiMCCollection() ) },
     _mcprimaryToken{ consumes<PrimaryParticle>(config().MCPrimary() ) }
-  {}
+  {
+    _station = config().ComboHitCollection().label() == "SFlagBkgHits";
```

Comment:
Note this logic will fail in the reconfigured sequence I'm working on, as the module instance will allways be FlagBkgHits, regardless of stereo or not.  I suggest instead you add a config parameter describing the expected level of the FlagBkgHits ComboHit collection (uniquepanel, etc ) and then test below on it matching.  Just copy how the config is setup for the outputlevel of FlagBkgHits.

Reply:
I see how that doesn't work. I was thinking for this check to happen automatically without requiring input from user. I guess this is easier, I'll change it.